src/gluonts/sagemaker/entry_point_scripts/run_entry_point.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Standard library imports
+import argparse
+import os
+import json
+import time
+
+
+def run(arguments):
+    print(time.strftime("%Y-%m-%d %H:%M:%S", time.gmtime()), "Starting - started running custom script.")
+    pass
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
```

Comment:
Argparse is what they use in the examples of their entrypoints, thats why I used it. It makes sense since its a standard library and does the job. Otherwise we would have to install click in the Docker environment to, increasing the Docker image size, and additionally we would further deviate from the MXNet container and SagemakerAPI.

Reply:
OK, so I put it on the TODO.